test/suites/expiration/suite_test.go

```diff
@@ -586,13 +586,6 @@ var _ = Describe("Expiration", func() {
 			// TODO: reduce timeouts when deprovisioning waits are factored out
 			env.EventuallyExpectNodesUntaintedWithTimeout(11*time.Minute, taintedNodes...)
 
-			// The nodeclaims that never registers will be removed
```

Comment:
Why are we removing this expectation vs increasing the timeout? Is the validation just not worth the additional E2E time or do the upstream changes break an assumption here that I'm not realizing?

Reply:
There's no change in upstream that breaks an assumption here. This expectation takes too long to succeed. In fact I have already increased the default time out to 16 minutes [here](https://github.com/aws/karpenter-provider-aws/pull/5979/files#diff-e50fbe8dc3a4e43a11436cbd3e0c86d0fefcb7713d7357cf67c5a566dcbca958R80). But the test would still fail when I tried running it on my local cluster with `[FAILED] Timed out after 960.000s.`